core/templates/services/context.service.spec.ts

```diff
@@ -160,6 +204,17 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that page contains reference to skills',
+      () => {
+        expect(ecs.getPageContext()).toBe('topic_editor');
+        expect(ecs.canEntityReferToSkills()).toBe(true);
+      });
+
+    it('should affirm the page context to be exploration player or editor',
```

Comment:
Is this description actually correct? We seem to be asserting that the context is *not* exploration.

Reply:
Done!